<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewSection</class>
 <widget class="QWidget" name="AddNewSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>521</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Комментарий к секции</string>
   </property>
   <widget class="QLineEdit" name="commentSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddCommentSection">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonWriteSectionInFile">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>160</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Записать секцию в файл</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>160</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Название секции</string>
   </property>
   <widget class="QLineEdit" name="nameSection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddNameSection">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="showAddedSection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>691</width>
     <height>520</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>140</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавленная секция</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>380</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Ключ и значение</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ключ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKey">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Значение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddInMap">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
